src/elements/element.line.js

```diff
@@ -52,56 +33,91 @@ function setStyle(ctx, vm) {
 	ctx.strokeStyle = vm.borderColor;
 }
 
-function normalPath(ctx, points, spanGaps, vm) {
-	const steppedLine = vm.steppedLine;
-	const lineMethod = steppedLine ? helpers.canvas._steppedLineTo : helpers.canvas._bezierCurveTo;
-	let move = true;
-	let index, currentVM, previousVM;
+function lineTo(ctx, previous, target) {
+	ctx.lineTo(target.x, target.y);
+}
+
+function getLineMethod(line) {
+	return line.steppedLine ? _steppedLineTo
```

Comment:
I think the nested ternary statements are hard to read here as well (and in the next method). Also, it'd be clearer to reverse the check and remove the `!`:
```
if (line.steppedLine) {
  return _steppedLineTo;
}
return line.tension ? _bezierCurveTo : lineTo;
```

Reply:
done